cmd/root.go

```diff
@@ -118,11 +119,15 @@ var RootCmd = &cobra.Command{
 			return ctx.NoContent(http.StatusOK)
 		})
 
-		r.Any("/metrics", echo.WrapHandler(promhttp.Handler()))
+		promRegistry := prometheus.NewRegistry()
+
 		// Register handler(s) for path(s)
-		tsl := tsl.NewTsl()
+		tsl := tsl.NewTsl(promRegistry)
```

Comment:
Instead of pass a custom promRegistry in the whole app, you can alter default registry
https://godoc.org/github.com/prometheus/client_golang/prometheus#pkg-variables

```go
prometheus.defaultRegistry = prometheus.NewRegistry
```